week-2/Homework/extra/1-radio-stations.js

```diff
@@ -25,8 +33,50 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function getStations() {
+  let AllFrequencies = [
+    87,
+    88,
+    89,
+    90,
+    91,
+    92,
+    93,
+    94,
+    95,
+    96,
+    97,
+    98,
+    99,
+    100,
+    101,
+    102,
+    103,
+    104,
+    105,
+    106,
+    107,
+    108,
+  ];
 
-
+  let stations = [];
+  for (let i = 0; i < AllFrequencies.length; i++) {
+    var isItRadioStation = isRadioStation(89);
```

Comment:
You already have isRadioStation function which checks if the frequency existe or not. Have a look to the solution and Understand if you have any question in this part feel free to ask.